tests/unit_tests/client/address_book/impl.cc

```diff
@@ -165,7 +209,9 @@ BOOST_AUTO_TEST_CASE(PGPClearSign) {
     "=P8Ug"
     "-----END PGP SIGNATURE-----";
   lines.push_back(line);
-  BOOST_CHECK(Validate());
+  //TODO(oneiric): PGP-signed subscriptions should pass validation
+  //  mark true after implementing PGP signature verification
```

Comment:
Wasn't sure if PGP signatures are actually verified by `AddressBook`. Thought it might be a good idea for validating the shipped `hosts.txt` + other subscriptions.

If `AddressBook` ignores PGP signatures in subscriptions, is PGP verification a feature somewhere else?

Reply:
>If AddressBook ignores PGP signatures in subscriptions, is PGP verification a feature somewhere else?

Yes, the reason for this is because our monero publisher service pulls the default subscription directly from github (see the pkg directory). I PGP sign the subscription to prevent any tampering; either by github or the very rare potential "I broke your TLS" mitm.